src/app/category/types.ts

```diff
@@ -0,0 +1,60 @@
+// export type MediaTypeMovie = 'movie';
+// export type MediaTypeTV = 'tv';
+// export type MediaTypePerson = 'person';
+// TODO: add types for media types in interfaces below
```

Comment:
There's no need to create type aliases, I should just use literals in interfaces